ui/src/Components/Filters/Filters.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useFilterContext } from '../../Context/filter_context';
+
+export default function Filters() {
+  const {
+    filterByEnabled,
+    filters: { status },
+    updateFilters,
+  } = useFilterContext();
+
+  const handleValueChange = (event) => {
```

Comment:
the name should be consistent with the rest of the `event` handlers. Instead of `handleValueChange` use `onValueChange` or something more align to the event happening.

Reply:
renamed as per suggestion